### PR TITLE
[native] Rename circleci setup script to centos

### PR DESCRIPTION
**Pull request by karteekmurthys**
Test plan - (Please fill in how you tested your changes)
No testing required. This a documentation update.

General Changes
1. Renamed scripts/setup-circleci.sh to scripts/setup-centos.sh.
2. Fixed the path to centos/linux setup script for PrestoCPP setup. From this commit history the script was renamed but the README is not updated to reflect the new name.
https://github.com/facebookexternal/presto_cpp/commits/main/scripts/setup-circleci.sh

```
== NO RELEASE NOTE ==
```

### CODE REVIEW

**Review by kgpai — APPROVED** [review]

**Review by ajaygeorge — APPROVED** [review]
LGTM.
I don't think this requires Release notes. You can remove that part.

File: presto-native-execution/README.md
Comment (mbasmanova) [inline presto-native-execution/README.md:112]: Is this the right fix? Shouldn't the script name be setup-ubuntu.sh ? CC: @spershin

**Review by mbasmanova — DISMISSED** [review]
@karteekmurthys Would you explain why is this the right fix?

**Comment by mbasmanova** [comment]
Here is a relevant PR: #18273

File: presto-native-execution/README.md
Comment (karteekmurthys) [reply presto-native-execution/README.md:112]: From this commit history, the setup-linux.sh was renamed to setup-circleci.sh.
https://github.com/facebookexternal/presto_cpp/commits/main/scripts/setup-circleci.sh

Comment (mbasmanova) [reply presto-native-execution/README.md:112]: CC: @mshang816 @spershin @majetideepak Should we rename setup-circleci.sh to setup-linux.sh to avoid confusion? It is strange to run setup-circleci.sh script in local environments.

Comment (majetideepak) [reply presto-native-execution/README.md:112]: I think `setup-circleci.sh` is fine. `linux` is a general term used for various Linux distros (eg. `yum` vs `apt`). It can confuse users since we do not support all distros.

Comment (mbasmanova) [reply presto-native-execution/README.md:112]: Should we use a distro name then? setup-centos.sh?

Comment (majetideepak) [reply presto-native-execution/README.md:112]: yeah `setup-centos.sh` is perfect!

**Comment by majetideepak** [comment]
@karteekmurthys can you rename `setup-circleci.sh` and its references to `setup-centos.sh`? Thanks.

**Comment by karteekmurthys** [comment]
@majetideepak @mbasmanova 
I renamed the script and updated README.

**Comment by majetideepak** [comment]
@karteekmurthys I see other references to `setup-circleci.sh` in `scripts/circleci-container.dockfile`. Can you scan other references again?

**Review by majetideepak — APPROVED** [review]

**Comment by majetideepak** [comment]
@karteekmurthys can you rebase with the latest master to get the test native passing?

**Comment by majetideepak** [comment]
@mbasmanova does this address your change request?

**Comment by mbasmanova** [comment]
> @mbasmanova does this address your change request?

The changes look great.

**Comment by kgpai** [comment]
As this is prestodb , will leave it to @ajaygeorge to merge.

**Comment by ajaygeorge** [comment]
@kgpai , we need a Presto committer for the merge.
@tdcmeehan can you please help us here.

**Comment by majetideepak** [comment]
@kgpai Just curious as to why this change falls under prestodb? All the changes are in the presto-native-execution folder, right? So Velox code owners should be able to merge?

**Comment by tdcmeehan** [comment]
Yes, feel free to merge, as the change is contained entirely within Prestissimo.

**Pull request merged**
